pkg/lib/pb/model/protos/models.proto

```diff
@@ -1186,12 +1172,16 @@ message Membership {
     }
     
     enum PaymentMethod {
-        MethodCard = 0; // Stripe
+        MethodCard = 0;
         MethodCrypto = 1;
-        MethodApplePay = 2;
-        MethodGooglePay = 3;
-        MethodAppleInapp = 4;
-        MethodGoogleInapp = 5;
+        MethodInappApple = 2;
+        MethodInappGoogle = 3;
+    }
+
+    enum EmailVerificationStatus {
+        NotVerified = 0;
+        CodeSent = 1;
```

Comment:
after sending the erroneous code, there will be a status "NotVerified"?

Reply:
No. I have added more comments, please review.